Dockerfile

```diff
@@ -1,21 +1,113 @@
-FROM openjdk:11
+FROM alpine:3.8
 
-ENV ES_VERSION="6.5.1"
+# Here we install GNU libc (aka glibc) and set C.UTF-8 locale as default.
+
+ENV LANG=C.UTF-8 \
+    JAVA_VERSION_MAJOR=8 \
```

Comment:
Why revert to java 8? Consider using https://hub.docker.com/r/adoptopenjdk/openjdk11/tags

It should be similiar to your changes https://github.com/AdoptOpenJDK/openjdk-docker/blob/master/11/jdk/alpine/Dockerfile.hotspot.releases.slim

Reply:
iirc something came after the mktemp issue that we couldn't solve... elastic put their hands up and pointed at the compatibility matrix... ill try and find out what it was... and 6.5.x supports java 11

Reply:
Yeah https://www.elastic.co/support/matrix#matrix_jvm, then we could use `openjdk10:jdk-10.0.2.13-alpine-slim`, as java 10 has better container support